src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -363,10 +385,13 @@ private function createFirewall(ContainerBuilder $container, $id, $firewall, &$a
         // Authentication listeners
         list($authListeners, $defaultEntryPoint) = $this->createAuthenticationListeners($container, $id, $firewall, $authenticationProviders, $defaultProvider, $configuredEntryPoint);
 
+        $config->replaceArgument(6, $configuredEntryPoint ?: $defaultEntryPoint);
```

Comment:
missing

```php
if (isset($firewall['anonymous'])) {
    $listenerConfigs['anonymous'] = $firewall['anonymous'];
}
```

for the `anonymous` listener registered in `createAuthenticationListeners()` ?